GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/Able/RO_Able.cfg

```diff
@@ -0,0 +1,179 @@
+//	================================================================================
+//	Able Configs
+//	================================================================================
+
+
+// Thor Able Avionics
+@PART[bluedog_ThorAble_Guidance]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@title = Thor Able Guidance Unit
+	@manufacturer = Aerojet
+	@description = Guidance core for the Thor Able upper stage as used on Thor Able and early Delta rockets. Goes on top of the Able second stage tank.
+	@mass = 0.11
+	@MODULE[ModuleCommand]
+	{
+		@RESOURCE[ElectricCharge]
+		{
+			@rate = 0.05
+		}
+	}
+	@MODULE[ModuleSAS]
+	{
+		%SASServiceLevel = 1
+	}
+	!MODULE[ModuleFuelTanks] {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 1
+		basemass = -1
+		type = ServiceModule

```

Comment:
This tank type apparently cannot hold helium, which results in it being literally incapable of carrying a fuel load for the AJ10.

Reply:
This is the avionics part, not the fuel tank itself, it just needs ElectricCharge